lib/hyperdrive_types/hyperdrive_types/ERC20MintableContract.py

```diff
@@ -23,6 +19,8 @@
 class ERC20MintableAllowanceContractFunction(ContractFunction):
     """ContractFunction for the allowance method."""
 
+    # super() call methods are generic, while our version adds values & types
+    # pylint: disable=arguments-differ
```

Comment:
maybe we should just move this comment & disable to the top?

Reply:
I actually do like it here, even though its repeated, so we can identify other areas this might show up.  same for other lint issues unless they really are globally annoying.